independent-projects/tools/devtools-common/src/main/java/io/quarkus/platform/tools/ToolsUtils.java

```diff
@@ -39,6 +40,44 @@ public static String getProperty(String name, String defaultValue) {
         return System.getProperty(name, defaultValue);
     }
 
+    public static boolean nonBlank(String arg) {
+        return !isBlank(arg);
+    }
+
+    public static boolean isBlank(String arg) {
+        return arg == null || arg.trim().isEmpty();
+    }
+
+    public static Map<String, String> stringToMap(
+            String str, String entrySeparator, String keyValueSeparator) {
+        try {
```

Comment:
If I remember right, you should be able to use StringUtils from apache commons, it contains helps to split and also `isBlank` and so on... @gastaldi right?

Reply:
I see, let me know if I can include the dependency. Also split is from String itself. Do you prefer to remove the rest of validations and just thrown the exception ? I can revert the impl to the stream.

Reply:
I guess it's alright like you did. I thought StringUtils would be more helpful to get a map..

Reply:
@ia3andy done, I'm working on tests, could you share how to run those tests ? I'm trying to run them unsuccessfully e.g
mvn test -f integration-tests/maven -Dtest=testProjectGenerationFromScratchWithAppConfigParameter

Same for unit tests. Thanks